src/types/plugin-input.ts

```diff
@@ -0,0 +1,44 @@
+import { SupportedEvents, SupportedEventsU } from "./context";
+import { StaticDecode, Type as T } from "@sinclair/typebox";
+
+export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU extends SupportedEvents[T] = SupportedEvents[T]> {
+  stateId: string;
+  eventName: T;
+  eventPayload: TU["payload"];
+  settings: StartStopSettings;
+  authToken: string;
+  ref: string;
+}
+
+const oneDay = 24 * 60 * 60 * 1000;
+
+export const startStopSchema = T.Object({
+  disabledCommands: T.Array(T.String(), { default: ["start"] }),
+  labels: T.Object(
+    {
+      time: T.Array(T.String()),
+      priority: T.Array(T.String()),
+    },
+    {
+      default: {
```

Comment:
Why are the labels relevant to the `start | stop`?

Reply:
in `get-time-labels-assigned.ts` used to calc the `duration` part of the metadata and also in `analytics.ts` which is used in `get-multiplier-info.ts` which in part of the comment.

```ts
  const timeLabelsDefined = config.labels.time;
  ...
   const timeLabel = timeLabelsDefined.find((label: string) => label === _labelName);
    if (timeLabel) {
      timeLabelsAssigned.push(_label);
    }
```

Reply:
I get it but why should they be defined within the configuration I meant? They should just be parsed. Because the configuration from the other plugin responsible for the labels can also be changed. But you can be guaranteed to get something looking like `Time: <X DURATION` with X being a number and duration `hour | day | week`

Reply:
I saw what you meant once I had another look at it so it's just parsing the labels from the issue itself now